_includes/title.html

```diff
@@ -19,10 +19,14 @@
     <h1>
         {{ page.title | xml_escape }}
         <small>{{ page.subtitle }}</small>
+        
         {% if edit_path %}
             <a class="btn btn-primary pull-right" href="{{ edit_path }}">Edit on Github</a>
         {% elsif show_news_link %}
             <a class="btn btn-primary pull-right" href="/news/atom.xml">Subscribe via Atom</a>
         {% endif %}
     </h1>
+    {% if page.author %}
```

Comment:
Why not just just create a new layout? For example, it should be fairly trivial to copy `_layouts/docs.html` and replace the contents with an author `<div>`. You just have to change <https://github.com/ev3dev/ev3dev.github.io/blob/master/_config.yml#L36> from `pages` to `tutorial` or whatever you name the new layout.